src/app/extensions.h

```diff
@@ -21,6 +21,10 @@ namespace ui {
   class Widget;
 }
 
+namespace {
+  const char* kAsepriteDefaultThemeExtensionName = "aseprite-theme";
+}
```

Comment:
Please don't use anonymous namespaces in header files. Move this member inside `Extension` class if needed.